http/http-minimum/src/test/java/io/quarkus/ts/http/minimum/DevModeHttpMinimumIT.java

```diff
@@ -69,4 +69,21 @@ public void shouldDetectChanges() {
                 () -> app.given().get("/hello").then().statusCode(HttpStatus.SC_OK)
                         .body("content", is(String.format(HELLO_IN_SPANISH, WORLD))));
     }
+
+    @Test
+    @Tag("QUARKUS-2112")
+    public void verifyWarningLogWhenBuildPropertyIsUpdated() {
+        String appName = app.getProperty("quarkus.application.name", "test-http");
+        String newAppName = "new-test-app";
+        app.given()
+                .param("action", "updateProperty")
+                .param("name", "quarkus.application.name")
+                .param("value", newAppName)
+                .post("/q/dev/io.quarkus.quarkus-vertx-http/config")
+                .then().statusCode(200);
+
+        String expectedOutput = String.format("quarkus.application.name is set to '%s' but it is build time fixed to '%s'",
+                appName, newAppName);
```

Comment:
Is it just me, or should the message be formulated the other way around?
I mean, the test passes, but the message seems wrong to me. It should be e.g.
```
quarkus.application.name is set to 'new-test-app' but it is build time fixed to 'test-http'
```
There was a discussion about it in the GH issue (https://github.com/quarkusio/quarkus/issues/23573#issuecomment-1088517560) making the exact same point, which the second PR (https://github.com/quarkusio/quarkus/pull/25512) was supposed to fix.

Reply:
Updated!, thanks!